tools/run_tests/run_performance_tests.py

```diff
@@ -433,7 +493,13 @@ def finish_qps_workers(jobs):
 if not args.dry_run:
   build_on_remote_hosts(remote_hosts, languages=[str(l) for l in languages], build_local=build_local)
 
-qpsworker_jobs = create_qpsworkers(languages, args.remote_worker_host)
+perf_cmd = None
+if args.perf_args:
+  # Expect /usr/bin/perf to be installed here, as is usual
+  perf_cmd = ['/usr/bin/perf'] 
+  perf_cmd.extend(args.perf_args.split(' '))
```

Comment:
if I put two spaces in the perf_args, will that result in and empty items in the list after splitting?   
this is a bit errorprone